<!-- 
  Launch sensor processor
  - Nodes for multi-sensor processing
-->

<launch>
  <arg name="debug_mode" default="false" />

  <node pkg="height_mapping" type="sensor_processor_node" name="sensor_processor" output="screen" />

</launch>